torus.msh

```diff
@@ -0,0 +1,700 @@
+$MeshFormat
```

Comment:
don't add meshes to the repo. use the data repo

Reply:
removed